apps/web/src/components/nav/NavMenuButton/NavMenuToggleButton.tsx

```diff
@@ -1,36 +1,55 @@
 import { FC, PropsWithChildren, useState } from 'react';
-import { Text } from '@novu/novui';
 import { css, cx } from '@novu/novui/css';
 import { Flex, HStack } from '@novu/novui/jsx';
 import { INavMenuButtonProps, rawButtonBaseStyles } from './NavMenuButton.shared';
 import { IconKeyboardArrowDown, IconKeyboardArrowUp } from '@novu/novui/icons';
+import { NavLink } from 'react-router-dom';
 
-type INavMenuToggleButtonProps = Omit<INavMenuButtonProps, 'rightSide'>;
+type INavMenuToggleButtonProps = { link: string } & Omit<INavMenuButtonProps, 'rightSide'>;
 
 export const NavMenuToggleButton: FC<PropsWithChildren<INavMenuToggleButtonProps>> = ({
   testId,
   icon,
   label,
   children,
   className,
+  link,
 }) => {
   const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [isHovered, setIsHovered] = useState<boolean>(false);
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** ‏
```suggestion
  const { isHovered, ...hoverProps } = useHover();
```